src/game_api/state.hpp

```diff
@@ -84,7 +90,13 @@ struct StateMemory
     Items* items;
     LevelGenSystem* level_gen;
     Layer* layers[2];
-    char pad12d0[0x1308 - 0x12d0];
+    size_t unknown23;
+    size_t unknown24;
+    size_t unknown25;
+    size_t unknown26;
+    size_t particle_emitters_info;

```

Comment:
Is this a pointer?

Reply:
It is, but I haven't investigated any deeper, and just needed this base pointer to give to the generate particles function. It's a pointer containing lots of pointers, several layers deeps, looks like they are maybe grouping particle emitters in sections or something. Will look into it more later.

Reply:
That's fine, I was just curious  :)